### ISSUE TITLE
Add source encoding support

### ISSUE THREAD

**Issue by gslowikowski** [scoverage/scoverage-maven-plugin#29]
Version `1.2.0` of `scalac-scoverage-plugin` adds source encoding support.

In Maven the convention is to specify source encoding as project property
```xml
<properties>
    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
</properties>
```

This plugin's new `encoding` configuration parameter will use this property as default value.
If no encoding is specified, platform encoding will be used (not recommended, see https://maven.apache.org/general.html#encoding-warning).

**Comment by gslowikowski** [comment]
New `1.2.0-SNAPSHOT` snapshot deployed.

**Comment by gslowikowski** [comment]
Documentation generated and [published](http://scoverage.github.io/scoverage-maven-plugin/1.2.0-SNAPSHOT/index.html).

**Comment by allenn** [comment]
Writing aggregate scenario is not fixed.

**Comment by gslowikowski** [comment]
Encoding support in aggregated reports fixed by #46, available since version `1.3.0`.
